eZ/Publish/Core/REST/Server/Controller/SessionController.php

```diff
@@ -163,11 +157,11 @@ public function deleteSessionAction($sessionId, Request $request)
      */
     private function hasStoredCsrfToken()
     {
-        if (!isset($this->csrfTokenStorage)) {
+        if (!isset($this->csrfTokenManager)) {
```

Comment:
Could you change it to `$this->csrfTokenStorage === null`? It is checked this way in other methods in this controller.